torchvision/prototype/datapoints/_video.py

```diff
@@ -32,27 +27,15 @@ def __new__(
         if data.ndim < 4:
             raise ValueError
         video = super().__new__(cls, data, requires_grad=requires_grad)
-
-        if color_space is None:
-            color_space = ColorSpace.from_tensor_shape(video.shape)  # type: ignore[arg-type]
-            if color_space == ColorSpace.OTHER:
-                warnings.warn("Unable to guess a specific color space. Consider passing it explicitly.")
-        elif isinstance(color_space, str):
-            color_space = ColorSpace.from_str(color_space.upper())
-        elif not isinstance(color_space, ColorSpace):
-            raise ValueError
-
-        return cls._wrap(tensor, color_space=color_space)
+        # TODO: Should this be `video` or can we remove it?
+        return cls._wrap(tensor)
```

Comment:
Yup, that seems like an artifact. Basically the `cls._to_tensor()` and `super().__new__` do the same thing. Let's remove the latter since it is obsolete.